addons/l10n_sk/data/account_tax_data.xml

```diff
@@ -3,7 +3,7 @@
     <!-- VAT domestic sale-->
     <record id="vy_tuz_20" model="account.tax.template">
         <field name="chart_template_id" ref="sk_chart_template"/>
-        <field name="name">DPH na výstupe 20%</field>
+        <field name="name">VAT on output 20%</field>
```

Comment:
Output? If we really want to add this information, I would use `20% S` or `20% Sales` etc.
cc @chklop

Reply:
Agreed